pipelines/XdawnCov_MDM.yml

```diff
@@ -0,0 +1,15 @@
+name: XdawnCovariances + MDM
+
+paradigms:
+  - P300
+
+citations:
+  - https://doi.org/10.1109/NER49283.2021.9441279
+  - https://citeseerx.ist.psu.edu/doc/10.1.1.713.5131
+
+pipeline:
+  - name: XdawnCovariances
+    from: pyriemann.estimation
```

Comment:
For XdawnCovariance, use xdawn_estimator='scm' and estimator='oas'

Reply:
Thanks! I've changed for XdawnCov_TS_SVM and xDAWN_LDA pipelines, too.

Reply:
perfect